portal-wire-protocol.md

```diff
@@ -213,6 +213,22 @@ selector = 0x02
 ssz-type = List[ByteList[2048], 32]
 ```
 
+
+##### Content encoding for uTP transfer
+
+In the case that the content item is send over uTP, it MUST be prefixed with a variable length unsigned integer (varint). The varint MUST hold the size, in bytes, of the consecutive content item.
+
+The varint encoding used is Unsigned LEB128.
+The maximum size allowed for this application is limited to `uint32`.
+
+The content item itself MUST be encoded as is defined for each specific network and content type.
```

Comment:
```suggestion
> The encoding of the content itself is specified by the content type being transferred.
```

Removing the MUST here feels like it might make this a little bit clearer.